install-dev/upgrade/sql/1.7.5.1.sql

```diff
@@ -0,0 +1,8 @@
+SET SESSION sql_mode='';
+SET NAMES 'utf8';
+
+ALTER TABLE `PREFIX_supply_order_receipt_history`
+  CHANGE `employee_lastname` `employee_lastname` VARCHAR(255) DEFAULT '';
+
+ALTER TABLE `PREFIX_product`
+  CHANGE `reference` `reference` varchar(64) DEFAULT NULL;
```

Comment:
Should it be supplier_reference when it was release with supplier_reference?

Reply:
The original intent was to change the size of both `reference` and `supplier_reference` from 32 to 64, not to rename it.